src/cli.rb

```diff
@@ -150,6 +150,19 @@ def run
         c.action Commands::Status
       end
 
+      command :ipmi do |c|
+        c.syntax = 'metal ipmi NODE_IDENTIFIER [options]'
+        c.summary = 'Perform ipmi commands on single or multiple machines'
+        c.description = "***VOLATILE***\n\n" \
+                    'Perform ipmi commands on single or multiple machines'
+        #c.example 'description', 'command example'
+        c.option '-g',
+          'Specifies that NODE_IDENTIFIER is the group. MUST be before NODE_IDENTIFIER'
```

Comment:
It isn't worth changing this in bash because it is probably faster to rewrite in ruby